src/data/partners.yml

```diff
@@ -0,0 +1,6 @@
+- name: "Tiktalik"
+  description: "Inexpensive virtual servers, Simple Cloud Computing, Cheap VPS"
+  country: "Poland"
+  city: "Warsaw"
+  pictureUrl: public/img/logo/tiktalik-logo.png
```

Comment:
Let's keep images in more domain friendly directories:
`img/people` for people ;),
`img/organizations` for all companies, organisations and such.

Also, please put FULL size images to the repo. The system resizes images during deployment process.